pkg/koyeb/utils.go

```diff
@@ -69,3 +72,29 @@ func loadYaml(file string) (string, error) {
 	}
 	return "", errors.New("Unknown format")
 }
+
+func watchDeployment(h *ServiceHandler, deploymentId string) {
+	numAttemptsOnUnhealthy := 3
```

Comment:
Your timer is too low, you should increase it to at least 2 minutes.

Reply:
done